tests/step-function/index.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const assert = require('assert');
+const sandbox = require('sinon').createSandbox();
```

Comment:
```suggestion
const sinon = require('sinon');
```

No hace falta crear otro sandbox. `sinon` ya es un sandbox.